Sources/Fuzzilli/FuzzIL/AbstractInterpreter.swift

```diff
@@ -201,7 +201,13 @@ public struct AbstractInterpreter {
         return result
     }
     
-    private mutating func executeEffects(_ instr: Instruction) {
+    private mutating func executeEffects(_ instr: Instruction, with runtimeType: (Variable, Type)?) {
```

Comment:
I'm not 100% happy about this API since the `Variable` parameter seems redundant, it's already in `instr.output`, no? But see also main comment

Reply:
The pair is there so we don't reimplement same logic twice https://github.com/googleprojectzero/fuzzilli/pull/128/files#diff-b12fd4c6f60ba74a8c5ba07622efca8dR63, becasue it can be either `output` or `innerOutput`